tests/ci/stress.py

```diff
@@ -47,6 +47,7 @@ def get_options(i: int, upgrade_check: bool) -> str:
 
     if i > 0 and random.random() < 1 / 3:
         client_options.append("use_query_cache=1")
+        client_options.append("query_cache_nondeterministic_function_handling='ignore'")
```

Comment:
This should not change anything, the real problem I guess is that `query_cache_system_table_handling` is required

Reply:
I wanted to keep the quoting of string parameters consistent with the ones above (e.g. l.36).

> the real problem I guess is that query_cache_system_table_handling is required

I did not see new new stress test runs in CI yet but yes, `query_cache_system_table_handling = ignore`  makes sense.

Reply:
Let's remove this quotation from the line below then? it only misleading developers :)

Reply:
OK, I will get back the quotation, to make the style common and to avoid superfluous changes.
This quotes can be removed later I guess...